src/components/FilterPanel.js

```diff
@@ -233,11 +233,12 @@ export default function FilterPanel(props) {
                             // Is the location within the range specified?
                             return d.miles < zipCodeFilter.miles;
```

Comment:
Also out of scope:

If I ask for locations within 10 miles, I think I want `loc <= 10 miles` not `loc < 10 miles`?